addons/website_slides/models/slide_slide.py

```diff
@@ -152,11 +152,21 @@ class Slide(models.Model):
     slide_views = fields.Integer('# of Website Views', store=True, compute="_compute_slide_views")
     public_views = fields.Integer('# of Public Views')
     total_views = fields.Integer("Total # Views", default="0", compute='_compute_total', store=True)
+    # comments
+    comments_count = fields.Integer('Number of comments', default=0, compute="_compute_comments_count", store=True)
+    # channel
+    channel_type = fields.Selection(related="channel_id.channel_type", string="Channel type")
+    allow_comment = fields.Boolean(related="channel_id.allow_comment", string="Allows comment")
```

Comment:
Maybe rename to channel_allow_comment so that we know it comes from the channel ?